docs/serving/creating-domain-mappings.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Creating a Mapping between a Custom Domain Name and a Knative Service (Alpha)"
+linkTitle: "Creating Domain Mappings (Alpha)"
+weight: 64
+type: "docs"
+---
+
+If the Domain Mapping feature is installed, you can serve
+custom domains backed by Knative Services. For example, if you
+own the "MyDomain.com" domain name, you can point DNS at your
+Knative cluster, and have this domain name be served by a
+Knative Service.
+
+## Before you begin
+
+To map a custom domain to a Knative Service, you should first [create a Knative
+Service](https://knative.dev/docs/serving/creating-services/).
+
+## Creating a Domain Mapping
+
+To create a mapping from a custom domain to a Knative Service, you need to
+create a YAML file that defines a Domain Mapping.
+This YAML file specifies the domain name to map and the Knative Service to use
+to service requests.
```

Comment:
Put some reference to the example “mydomain.com” explain if using this value would cover all sub domains ? *.mydomain.com or it needs to a specific domain ?

Reply:
added an extra para here, ptal